internalbindings/internalbindings.go

```diff
@@ -75,6 +75,8 @@ type (
 	TimerID = internal.TimerID
 	// ContinueAsNewError used by a workflow to request continue as new
 	ContinueAsNewError = internal.ContinueAsNewError
+	// UpdateCallbacks used by a workflow to request updates
```

Comment:
This comment isn't correct, the callbacks  is used to report the result of an update,

Reply:
Fixed